recipes/default.rb

```diff
@@ -291,7 +291,8 @@
          :onlinefs_password => encrypted_onlinefs_password,
          :onlinefs_salt => onlinefs_salt,
          :pki_ca_configuration => caConf.to_json(),
-         :usernames_configuration => usernamesConfiguration.to_json()
+         :usernames_configuration => usernamesConfiguration.to_json(),
+         :kafka_installed => exists_local("kkafka", "default")
```

Comment:
You should have the `valid_recipe` as before because Kafka might not be on the same host as Hopsworks